core-tests/jvm/src/test/scala/zio/IOSpec.scala

```diff
@@ -144,26 +144,26 @@ class IOSpec(implicit ee: org.specs2.concurrent.ExecutionEnv) extends TestRuntim
   }
 
   def t6 = {
-    val list = List(1, 2, 3).map(IO.succeedLazy[Int](_))
+    val list = List(1, 2, 3).map(IO.succeed[Int](_))
     val res  = unsafeRun(IO.collectAllPar(list))
     res must be_===(List(1, 2, 3))
   }
 
   def t7 = {
-    val list = List(1, 2, 3).map(IO.succeedLazy[Int](_))
+    val list = List(1, 2, 3).map(IO.succeed[Int](_))
```

Comment:
I think it's better to replace most of these by `effectTotal` since otherwise the fiber workload becomes too trivial which may interfere with the original intention of the test writer...

Reply:
:heavy_check_mark: